src/app/_components/bottom-button.tsx

```
@@ -0,0 +1,43 @@
+  const router = useRouter();
+
+  const handleNavigateToAccessibilityQuiz = () => {
+    router.push(`${BROWSER_PATH.QUIZ.DEFAULT}?step=1`);
+  };
+
+  const handleNavigateToUpload = () => {
+    router.push(BROWSER_PATH.LABEL.UPLOAD);
+  };
+  return (
+    <nav
+      role='navigation'
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)